sunpy/data/test/erne/cr1907p.txt

```diff
@@ -0,0 +1,15 @@
+		Proton Intensities for Carrington Rot. 1907
+ 
+  Date   begin end Intensities [1/(cm^2*sr*s*MeV)] by energy channels [MeV]
+yy mm dd hhmm hhmm  1.8-3.3   3.3-6.4   6.4-12.7  13.5-25.8 25.8-50.7
+96 03 11 0000 0200  5.09E-03  9.79E-04  1.20E-04  1.20E-04  1.37E-04
+96 03 11 0200 0400  4.07E-03  8.74E-04  1.69E-04  1.47E-04  1.38E-04
+96 03 11 0400 0600  3.13E-03  5.82E-04  1.21E-04  1.16E-04  1.35E-04
+96 03 11 0600 0800  6.00E-03  7.28E-04  1.92E-04  1.28E-04  1.48E-04
+96 03 11 0800 1000  4.80E-03  7.27E-04  2.40E-04  1.24E-04  1.41E-04
+96 03 11 1000 1200  5.06E-03  8.73E-04  1.21E-04  1.20E-04  1.43E-04
+96 03 11 1200 1400  5.28E-03  6.80E-04  1.20E-04  1.26E-04  1.42E-04
+96 03 11 1400 1600  2.68E-03  1.08E-03  3.66E-04  1.27E-04  1.45E-04
+96 03 11 1600 1800  3.39E-03  8.36E-04  7.25E-05  1.22E-04  1.37E-04
+96 03 11 1800 2000  3.09E-03  7.79E-04  2.58E-05  1.31E-04  1.36E-04
+96 03 11 2000 2200  2.49E-03  8.04E-04  1.74E-04  1.45E-04  1.37E-04
```

Comment:
Disclaimer: I didn't create any of these files like specially. All I did was take original data files and delete the rows after a certain number of rows so that we have sufficient rows of data to check. This stands true for all the files.

I will accommodate the comments though.